packages/testing/README.md

```diff
@@ -169,3 +169,22 @@ Additionally you may need to set your CHROME_BIN env variable `export CHROME_BIN
 ::::
 
 For some example tests, please take a look at our [Set-Game Example Test Files](https://github.com/open-wc/example-vanilla-set-game/tree/master/test).
+
+
+## Fixture Cleanup
+By default if you import via `import { ... } from '@open-wc/testing';` it will include a side-effect that cleans up fixtures.
```

Comment:
```suggestion
By default, if you import anything via `import { ... } from '@open-wc/testing-helpers';`, it will automatically register a side-effect that cleans up your fixtures.
```